src/domain/signupGroup/signupGroupFormFields/signups/signup/Signup.tsx

```diff
@@ -71,93 +72,125 @@ const Signup: React.FC<Props> = ({
-              component={TextInputField}
-              disabled={disabled}
-              label={translateSignupText('labelLastName')}
-              placeholder={translateSignupText('placeholderLastName')}
-              required={isSignupFieldRequired(
-                registration,
-                SIGNUP_FIELDS.LAST_NAME
-              )}
-            />
-          </div>
-        </FormGroup>
-        <FormGroup>
-          <div className={styles.phoneNumberRow}>
-            <Field
-              name={getFieldName(signupPath, SIGNUP_FIELDS.PHONE_NUMBER)}
-              component={TextInputField}
-              disabled={disabled}
-              label={translateSignupText('labelPhoneNumber')}
-              placeholder={translateSignupText('placeholderPhoneNumber')}
-              required={isSignupFieldRequired(
-                registration,
-                SIGNUP_FIELDS.PHONE_NUMBER
+              {isSignupFieldRequired(registration, SIGNUP_FIELDS.LAST_NAME) && (
+                <Field
+                  name={getFieldName(signupPath, SIGNUP_FIELDS.LAST_NAME)}
+                  component={TextInputField}
+                  disabled={disabled}
+                  label={translateSignupText('labelLastName')}
+                  placeholder={translateSignupText('placeholderLastName')}
+                  required={true}
+                />
               )}
-            />
-          </div>
-        </FormGroup>
-        <FormGroup>
-          <div className={styles.streetAddressRow}>
-            <Field
-              name={getFieldName(signupPath, SIGNUP_FIELDS.STREET_ADDRESS)}
-              component={TextInputField}
-              disabled={disabled}
-              label={translateSignupText('labelStreetAddress')}
-              placeholder={translateSignupText('placeholderStreetAddress')}
-              required={isSignupFieldRequired(
+            </SplittedRow>
+          </FormGroup>
+        )}
+
+        {isSignupFieldRequired(registration, SIGNUP_FIELDS.PHONE_NUMBER) && (
+          <FormGroup>
+            <SplittedRow>
+              <Field
+                name={getFieldName(signupPath, SIGNUP_FIELDS.PHONE_NUMBER)}
+                component={TextInputField}
+                disabled={disabled}
+                label={translateSignupText('labelPhoneNumber')}
+                placeholder={translateSignupText('placeholderPhoneNumber')}
+                required={true}
+              />
+            </SplittedRow>
+          </FormGroup>
+        )}
+        {(isSignupFieldRequired(registration, SIGNUP_FIELDS.STREET_ADDRESS) ||
+          isDateOfBirthFieldRequired(registration)) && (
+          <FormGroup>
+            <SplittedRow
+              className={styles.streetAddressRow}
+              type={
+                isSignupFieldRequired(
+                  registration,
+                  SIGNUP_FIELDS.STREET_ADDRESS
+                )
+                  ? 'large-small'
```

Comment:
Same suggestion here: maybe these could be defined as constants and used from a constants file?

Reply:
Added `SPLITTED_ROW_TYPE` enum and `DEFAULT_SPLITTED_ROW_TYPE` constant to define type for the SplittedRow component